crossbar/router/dealer.py

```diff
@@ -1276,9 +1276,6 @@ def processYield(self, session, yield_):
                         if invocation_sent:
                             self._call_store.pop_queued_call(invocation_request.registration)
 
-        else:
-            raise ProtocolError(u"Dealer.onYield(): YIELD received for non-pending request ID {0}".format(yield_.request))
-
     def processInvocationError(self, session, error):
```

Comment:
I feel like we should at least log this (even if only at DEBUG level?)  in case there is an underlying problem.

Reply:
Yes. I can agree with that.